deployment/geonode/templates/pycsw/pycsw-cfg.yaml

```diff
@@ -0,0 +1,70 @@
+{{ if .Values.pycsw.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-pycsw-cfg
+  namespace: {{ .Release.Namespace }}
+data:
+  pycsw.cfg: |+
```

Comment:
Should we better include that configuration as is in the values.yml and embed the config via `{{- include "pycsw.config" . | nindent 4 }}`?